class.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable  max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const book = document.querySelector('.books');
+
+class Display {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') == null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static displayBooks() {
+    const books = Display.getBooks();
+    let display = '';
+    books.forEach((book, i) => {
+      display += `
+          <div class="booksAdded">
+          <p>"${book.title}" by ${book.author}</p>
+          <button class="removeBtn" onclick="Display.removeBook(${i})">Remove</button>
+          </div>`;
+    });
+    book.innerHTML = display;
+  }
+
+  static addBook() {
+    const titleInput = document.querySelector('#fname').value;
+    const authorInput = document.querySelector('#lname').value;
+    if (titleInput !== '' && authorInput !== '') {
+      const newBook = new Book(titleInput, authorInput);
+      const books = Display.getBooks();
+      books.push(newBook);
+      localStorage.setItem('books', JSON.stringify(books));
+      this.displayBooks();
+    }
+  }
+
+  static removeBook(id) {
+    const books = Display.getBooks();
+    const bookIndex = books.findIndex((item, i) => i === id);
+    books.splice(bookIndex, 1);
+    localStorage.setItem('books', JSON.stringify(books));
+    this.displayBooks();
+  }
+}
+window.addEventListener('DOMContentLoaded', () => {
+  Display.displayBooks();
+});
+const form = document.querySelector('.form');
+form.addEventListener('submit', (evt) => {
+  evt.preventDefault();
+  Display.addBook();
+  form.reset();
+});
```

Comment:
currently, when I try to add a book, it doesn't work and I get an error message on the console saying " Cannot read properties of null (reading 'value') at Display.addBook" kindly fix this issue as it is required that even as we refactor our code to classes, we should retain the functionality of the application👍👍👍